doc/keymap-format-text-v1.md

```diff
@@ -9,11 +9,15 @@ NOTE:
   syntax highlighting.
 -->
 
-This document describes the `XKB_KEYMAP_FORMAT_TEXT_V1` keymap format,
+This document describes the `XKB_KEYMAP_FORMAT_TEXT_V1` [keymap] format,
 as implemented by libxkbcommon.
 
-The standard database of keyboard configuration data is
-[xkeyboard-config].
+@see For an overview of the role of this format, please see “@ref xkb-the-config ""”.
+
+@see For examples of keymaps in this format, please see [xkeyboard-config], the
+standard database of keyboard configuration data.
+
+<!-- TODO quick guide for modify existing keymaps & creating custom ones -->
```

Comment:
We do have https://xkbcommon.org/doc/current/md_doc_user_configuration.html

Reply:
Good point! I think it would be useful to:
[ ] add links in this page that point to relevant sections in V1 format page
[ ] improve the main page introduce the library to different user profiles: users of the API and users that just want to know how to modify/add layouts.
[ ] create a FAQ page.